flow/designs/asap7/megaboom/constraint.sdc

```diff
@@ -1,6 +1,6 @@
 set clk_name      clock
 set clk_port_name clock
-set clk_period    5000 
+set clk_period    7500 
```

Comment:
Interesting.... I want to hear your thoughts on what going to 1000ps would take... fastest CPUs at 7nm are 200-333ps...

Reply:
This is just to get thru CTS + OPT + Routing...

We never have a session that is clean.

After we have a base, we will reduce the period.